lib/helpers.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var join = require('path').join;
+
+module.exports = {
+  expandHomeDir: function (path) {
+    var isWindows = process.platform == 'win32';
+    var homeDir = process.env[isWindows ? 'USERPROFILE' : 'HOME'];
+
+    if (!path) {
+      return path;
+    }
+
+    if (homeDir) {
+      if (path === '~') {
+        return homeDir;
+      }
+
+      if (path.slice(0, 2) !== '~/') {
+        return path;
+      }
+    } else {
```

Comment:
You could've done `if (!homeDir)` instead, and this `else` would not be as hidden down here, and it would also have reduced some indentation :)